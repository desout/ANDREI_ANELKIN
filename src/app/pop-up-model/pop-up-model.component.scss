.popUp{
  z-index: 20;
}
