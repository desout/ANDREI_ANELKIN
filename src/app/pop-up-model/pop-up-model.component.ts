import {Component, Input} from '@angular/core';
import {NgbActiveModal} from '@ng-bootstrap/ng-bootstrap';

@Component({
  selector: 'app-pop-up-model',
  templateUrl: './pop-up-model.component.html',
  styleUrls: ['./pop-up-model.component.scss']
})
export class PopUpModelComponent {
  @Input() name;
  constructor(public activeModal: NgbActiveModal) { }


}
