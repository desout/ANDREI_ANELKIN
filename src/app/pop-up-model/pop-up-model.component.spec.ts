import { async, ComponentFixture, TestBed } from '@angular/core/testing';

import { PopUpModelComponent } from './pop-up-model.component';
import {NO_ERRORS_SCHEMA} from '@angular/core';
import {TestModule} from '../Test.module';

describe('PopUpModelComponent', () => {
  let component: PopUpModelComponent;
  let fixture: ComponentFixture<PopUpModelComponent>;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      declarations: [],
      schemas: [NO_ERRORS_SCHEMA],
      imports: [TestModule],
    })
    .compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(PopUpModelComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(fixture.nativeElement).toMatchSnapshot();
  });
});
