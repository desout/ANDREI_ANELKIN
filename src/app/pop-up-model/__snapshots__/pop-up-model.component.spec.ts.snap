// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopUpModelComponent should create 1`] = `
<div
  id="root0"
  ng-version="7.0.3"
>
  <div
    class="popUp"
  >
    <div
      class="modal-header"
    >
      <h4
        class="modal-title"
      >
        INFO.POPUPHEADER
      </h4>
      <button
        aria-label="Close"
        class="close"
        type="button"
      >
        <span
          aria-hidden="true"
        >
          Ã—
        </span>
      </button>
    </div>
    <div
      class="modal-body"
    >
      <p>
        INFO.POPUPLOGIN !!!
      </p>
    </div>
    <div
      class="modal-footer"
    >
      <button
        class="btn btn-outline-dark"
        type="button"
      >
        INFO.POPUPCLOSE
      </button>
    </div>
  </div>
</div>
`;
